### PR TITLE
Coherence fixes

### PR DESCRIPTION
**Pull request by pranavkm**
cc @davidfowl cc @troydai

[Review on Reviewable](https://reviewable.io/reviews/aspnet/coherence/15)

### CODE REVIEW

File: makefile.shade [hunk 1]
```diff
@@ -34,7 +34,7 @@ use-standard-lifecycle
         {
             var versionsToDelete = Directory.EnumerateDirectories(package)
                                             .OrderByDescending(d => d, StringComparer.OrdinalIgnoreCase)
-                                            .Skip(1);
+                                            .Skip(3);
```
Comment (troydai) [inline makefile.shade]: Nice.

File: src/SanityCheck/Program.cs [hunk 2]
```diff
@@ -49,6 +49,7 @@ static int Main(string[] args)
                 "DNX-Darwin",
                 "DNX-Linux",
                 "Entropy",
+                "EventNotification",
```
Comment (troydai) [inline src/SanityCheck/Program.cs]: EventNotification is not for beta5. Can we skip this for release?

**Comment by troydai** [comment]
:watch:

File: src/SanityCheck/Program.cs (hunk 2 above)
Comment (pranavkm) [inline src/SanityCheck/Program.cs]: That's what this does

Comment (troydai) [inline src/SanityCheck/Program.cs]: I misread.

Comment (troydai) [inline src/SanityCheck/Program.cs]: Skip Localization as well?

**Comment by troydai** [comment]
Ping @davidfowl 

We need this to ensure CoreCLR packages are not partially wiped from latest-packages/release folder.

**Pull request merged**